data/categories.txt

```diff
@@ -6484,6 +6484,88 @@ sw:Malazi
 fa:پناهگاه
 mr:निवारा
 
+amenity-shelter-basic_hut
+en:3Bivouac Box
+ar:4صندوق إقامة مؤقتة
+be:4Бівачны бокс
+bg:4Кутия за бивак
+ca:4Caixa bivac
+cs:4Bivakovací box
+da:4Bivuak-kasse
+de:4Biwakschachtel
+el:4Κιβώτιο Bivouac
+es:4Caja vivac
+et:4Bivouac box
+eu:4Bivouac kutxa
+fa:4جعبه بیواک
+fi:4Bivouac-laatikko
+fr:4Boîte de bivouac
+he:4קופסת ביבוק
+hi:4बिवौक बॉक्स
+hu:4Bivak doboz
+id:4Kotak bivak
+it:4Bivacco|Box da bivacco
+ja:4ビバークボックス
+ko:4비박 상자
+mr:4बिव्होक बॉक्स
+nb:4Bivuakkasse
+nl:4Bivakdoos
+pl:4Skrzynka biwakowa
+pt:4Caixa de bivaque
+ro:4Cutia de bivuac
+ru:4Бивуачный ящик
+sk:4Bivakovací box
+sv:4Bivackbox
+sw:4Sanduku la Bivouac
+th:4กล่องพักแรม
+tr:4Bivak kutusu
+uk:4Бівуачна будка
+vi:4Hộp bivouac
+zh-Hans:1宿营箱
+zh-Hant:1露營箱
+
+amenity-shelter-lean_to
+en:3Lean-to Shelter
+ar:العجاف إلى المأوى
+be:Навесны прытулак
+bg:Заслон Lean-to
+ca:Refugi inclinat
+cs:Přístřešek Lean-to
+da:Læskur
+de:Lean-to Schutzhütte
+el:Καταφύγιο με κλίση προς τα πάνω (Lean-to Shelter)
```

Comment:
Please, no ..

Reply:
A good solution would be to clearly write about keywords limitations at the top of the file.

Reply:
What limitation? Even if we assume that we have the very best matching algo, who will be writing this string to search for the category? "Καταφύγιο με κλίση προς τα πάνω (Lean-to Shelter)"